### PR TITLE
Chore/v2 docs

### PR DESCRIPTION
**Pull request by kiwicopple**

### CODE REVIEW

File: apps/reference/_storage/intro.mdx [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+slug: /
+sidebar_position: 1
+sidebar_label: Introduction
```
Comment (dannykng) [inline apps/reference/_storage/intro.mdx:4]: Should we use `Introduction` or the name of the reference (e.g., https://reference-docs-git-chore-v2docs-supabase.vercel.app/cli)?

Screen Shot 2022-08-02 at 4 05 49 PM

File: apps/reference/nav/storage_sidebars.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+ */
+
+// @ts-check
+
+/** @type {import('@docusaurus/plugin-content-docs').SidebarsConfig} */
+const sidebars = {
+  // By default, Docusaurus generates a sidebar from the docs folder structure
+  // tutorialSidebar: [{type: 'autogenerated', dirName: '.'}],
+
+  goTrueSidebar: [
```
Comment (dannykng) [inline apps/reference/nav/storage_sidebars.js:19]: Should this be something like `storageSidebar` since this file is `storage_sidebars.js`?

**Review by dannykng — APPROVED** [review]
Couple other things:
- The carets in the sidebar are misaligned
Screen Shot 2022-08-02 at 4 13 41 PM
- Should the Supabase API ref be collapsible? This is the only ref that has it.
Screen Shot 2022-08-02 at 4 14 41 PM

Nothing blocking so feel free to merge if you want to start a new branch to continue iterating!

File: apps/reference/_api/usage.mdx [hunk 3]
```diff
@@ -14,50 +13,37 @@ An isomorphic Javascript library for GoTrue.
...
 
-## New user signup
 
-```js
-const { user, session, error } = await gotrue.signUp({
-  email: 'example@email.com',
-  password: 'example-password',
-})
-```
+### Lists all organizations
```
[the remark this answers is not in the document]
Comment (kiwicopple) [reply apps/reference/_api/usage.mdx:17]: This one is required to be updated on the API itself (i already updated the API team to do it)

File: apps/reference/nav/_referenceSidebars.js [hunk 4]
```diff
@@ -27,34 +27,36 @@ const sidebars = {
...
-      link: { type: 'doc', id: 'postgres/extensions/intro' },
-      collapsed: true,
+      label: 'Self hosted',
+      collapsed: false,
       items: [
-        'postgres/extensions/http',
-        'postgres/extensions/pgtap',
-        'postgres/extensions/plv8',
-        'postgres/extensions/uuid-ossp',
+        { type: 'link', label: 'Auth Server', href: '/gotrue' },
```
[the remark this answers is not in the document]
Comment (kiwicopple) [reply apps/reference/nav/_referenceSidebars.js:44]: I agree - I would be happy to move to "auth" but it's a bit too much work for right now 😂

**Pull request merged**